<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUICreateNewView</class>
 <widget class="QDialog" name="GUICreateNewView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>View Name</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="viewName"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="viewType">
        <item>
         <property name="text">
          <string>COMPONENT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NODE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EDGE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FACE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SUBSYSTEM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RASTERDATA</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_modifyStream">
        <property name="text">
         <string>Modify existing view</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_startNewStream">
        <property name="text">
         <string>Start new stream</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Attributes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListWidget" name="attributeList"/>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>New attribute</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QLineEdit" name="newAttributename"/>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="newAttributeType">
                 <item>
                  <property name="text">
                   <string>DOUBLE</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>STRING</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>TIMESERIES</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>LINK</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>DOUBLEVECTOR</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>STRINGVECTOR</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="buttonAddAttribute">
                 <property name="text">
                  <string>Add Attribute</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Delete attribute</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Select attributes in the Attributes list and click delete.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="1">
                <widget class="QPushButton" name="buttonDeleteAttribute">
                 <property name="text">
                  <string>Delete Attribute</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GUICreateNewView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GUICreateNewView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
